package.json

```diff
@@ -55,7 +55,8 @@
   },
   "resolutions": {
     "encoding": "0.1.13",
-    "expect-playwright": "0.8.0"
+    "expect-playwright": "0.8.0",
```

Comment:
We have to force an older version of this dependency because it includes playwright. Later versions of playwright require Jest 28 which isn't in React Scripts. I should write documentation